pkg/distro/rhel/rhel10/options.go

```diff
@@ -81,11 +81,13 @@ func checkOptions(t *rhel.ImageType, bp *blueprint.Blueprint, options distro.Ima
 		warnings = append(warnings, w)
 	}
 
-	if customizations.GetInstaller() != nil {
+	if instCust, err := customizations.GetInstaller(); instCust != nil {
```

Comment:
Same idea/suggestion as before, I would do the error check first here to make it more linear.